### DISCUSSION TITLE
Default ACL values

### DISCUSSION THREAD

**Discussion by db0** [django-wiki/django-wiki#1307] [Q&A] [answered]
I am reading the settings and I see values like `wiki.conf.settings.CAN_ADMIN = None`, which implies there's a default True/False, but that default is never mentioned

**Comment by benjaoming** [comment]
![image](https://github.com/django-wiki/django-wiki/assets/374612/c88b0af5-9952-4cca-8f39-243bf4ef0b1c)

This can supply a callable, so it is more complex than True/False.

**Reply by db0** [reply]
I see. Where would the function be picked up from?

**Reply by benjaoming** [reply, marked as answer]
Let's continue the discussion here: https://github.com/django-wiki/django-wiki/discussions/1308#discussioncomment-7151250
